events/member.py

```diff
@@ -1,10 +1,8 @@
-import inspect
```

Comment:
This is required, you cannot remove this

Reply:
okay i added it again